source/get-started/pojo-qs.txt

```diff
@@ -0,0 +1,410 @@
+
+.. important::
+
+   When querying POJOs you *must* query against the document field name
+   and not the POJO property name. By default, they are the same but it
+   is possible to change how POJO property names are mapped.
+
+Get A Single Person That Matches a Filter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For example, to find the first ``Person`` in the database that lives in
+``Wimborne``, pass an ``eq()`` filter object to specify the equality
+condition:
+
+.. code-block:: java
+
+   collection.find(eq("address.city", "Wimborne"))
+       .first()
+       .subscribe(new PrintToStringSubscriber<>());
+
+The example prints one document:
+
+.. code-block:: none
+
+   Person{id='591dbc2550852fa685b3ad1a', name='Timothy Berners-Lee', age=61, address=Address{street='Colehill', city='Wimborne', zip='null'}}
+
+Get All Person Instances That Match a Filter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example prints every document in which the value of
+``age`` is greater than ``30``:
+
+.. code-block:: java
+
+   collection.find(gt("age", 30)).subscribe(new PrintToStringSubscriber<>());
+
+Update Documents
+----------------
+
+To update documents in a collection, you can use the collection's
+``updateOne()`` and ``updateMany()`` methods.
+
+Pass the following components to the methods:
+
+- Filter object to determine the document or documents to update. To
+  specify an empty filter and match all ``Person`` instances, use an
+  empty ``Document`` object.
+- An update document that specifies the modifications. To view a list of
+  the available operators, see :manual:`Update Operators
+  </reference/operator/update/>` in the Server manual.
+
+The update methods return an ``UpdateResult`` type which provides
+information about the operation, including the number of documents
+modified by the update.
+
+Update a Single Person
+~~~~~~~~~~~~~~~~~~~~~~
+
+To update a single ``Person``, use the ``updateOne()`` method.
+
+The following example updates a ``Person`` named ``"Ada Byron"`` by
+setting their age to ``23`` and name to ``Ada Lovelace``:
+
+.. code-block:: java
+
+   collection.updateOne(
+       eq("name", "Ada Byron"), 
+       combine(set("age", 23), set("name", "Ada Lovelace"))
+   ).subscribe(new OperationSubscriber<>());
+
+Update Multiple Person Instances
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To update all ``Person``  instances that match a filter, use the
+``updateMany()`` method.
+
+The following example sets the ``zip`` field to ``null`` for all documents
+that have a ``zip`` value:
+
+.. code-block:: java
+
+   collection.updateMany(not(eq("zip", null)), set("zip", null))
+            .subscribe(new OperationSubscriber<>());
+
+Replace a Single Person
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Another method to change an existing ``Person`` instance is to use
+the ``replaceOne()`` method.
+
+The following example replaces a ``Person`` named ``"Ada Lovelace"``
+with the original ``Person`` instance:
```

Comment:
S: Instead of saying "original Person instance," it would be helpful to link to the section that defines that original instance:

```suggestion
The following example replaces a ``Person`` named ``"Ada Lovelace"``
with the ``Person`` instance, ``ada``, defined in the :ref:`<link to line 210>` section of this guide:
```